lib/matplotlib/colors.py

```diff
@@ -1731,7 +1731,7 @@ def __call__(self, value, clip=None):
         else:
             max_col = self.Ncmap
         # this gives us the bins in the lookup table in the range
-        # [0, _n_regions - 1]  (the offset is baked in the init)
+        # [0, _n_regions - 1]  (the offset is baked-in in the init)
```

Comment:
```suggestion
        # [0, _n_regions - 1]  (the offset is set in the init)
```
can be merged without a change, but baked-in feels too idiomatic